lib/tupelo.js

```diff
@@ -628,20 +628,6 @@ class TupeloClient {
    * @property {StorageAdapterConfigForIpld} ipld - Configure and use the ipld storage adapter
    */
 
-  /**
-   * @typedef {Object} SerializedChainTree
```

Comment:
there is a protobuf of SerializedChainTree is that not used?

Reply:
I didn't see `SerializedChainTree` anywhere --- `SerializableChainTree` does exist, but it is never directly returned from any of our methods. That being said, it is the result of decoding the base64 string returned from `exportChainTree`, so maybe it's worth documenting (though still a completely different typedef than above)

Reply:
oh maybe I'm remembering the past :)

Reply:
ah yes - I was thinking "https://github.com/quorumcontrol/messages/blob/master/src/services/services.proto#L40"